modules/hrm/views/requests.php

```diff
@@ -80,12 +40,12 @@
                                 class="wperp-btn btn--cancel btn--filter"
                                 @click.prevent="toggleDropdown()"
                                 value="<?php _e( 'Cancel', 'erp' ); ?>">
-                            
```

Comment:
Same extra space issue also in this whole file. If possible to remove, please remove.

Reply:
Same thing applies here. I didn't find any white space in empty line.